src/com/wsscode/pathom/connect.cljc

```diff
@@ -579,9 +579,9 @@
   (and (every? (fn [[_ v]] (not (p/break-values v))) m)
        (every? m input)))
 
-(defn- cache-batch [env resolver-sym linked-results]
+(defn- cache-batch [env resolver-sym p linked-results]
```

Comment:
this is a breaking change without compatible backward version, so this a big no no.

in this case, this new `p` param is not even needed, since you already get the `env` here the params can be extracted directly from it, so the line `584` here  is the only place that actually needs change, all the rest can be just deleted I think.

Reply:
Sorry for the breaking change! Since it was a private function, I didn't anticipate it being an issue. Is it specifically the signature of `cache-batch` changing that is the problem?

I believe the other changes might be necessary so that the (already cached) response isn't shared across different parameter sets. ie. if `795`, `865`, `984` etc was missing, wouldn't the same response be used for the same queries that have different parameters?

I am definitely not confident in my code, just walking through my understanding of it.

Reply:
What I'm suggesting is just that you can extract the param from the env in the `cache-batch` fn and use it. Right now in the changes, you are extracting params from env in multiple points to send here, but you already have that data in the `env` (you can check, the same env you extracted the params is sent here). Makes sense?